backend/src/core/schemas/user/base.py

```diff
@@ -41,8 +43,12 @@ class UserCreate(UserBase):
     viber_link: str | None = None
     telegram_link: str | None = None
     instagram_link: str | None = None
-    theatrical_role: list[int] | None = None
 
 
-class UserUpdate(UserCreate):
-    password: str | None = None
+class UserUpdate(BaseModel):
+    first_name: Optional[str]
+    last_name: Optional[str]
+    email: Optional[str]
+    free_dates: Optional[list[str] | None] = None
+    phone_number: Optional[str]
+    theatrical_role: Optional[object]  # need to implement
```

Comment:
WTF?
У нас же typing в питоне 10 завезли крутой, зачем это делать?

```suggestion
class UserUpdate(BaseModel):
    first_name: str | None = None
    last_name: str | None = None
    email: str | None = None
    free_dates:list[str] | None = None
    phone_number: str | None = None
    theatrical_role: ТУТ ДОЛЖНА БЫТЬ pydantic модель theatrical_role | None = None
```